### PR TITLE
Add 2021-04-27 challenge

### PR DESCRIPTION
**Pull request by parhaml**
Challenge for 4/27/2021

### CODE REVIEW

File: challenges/2021-04-27-sum-arrays/challenge.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Given an array of values, return the sum of the values.
+
+## Business Rules/Errata
+
+- Input must be an array.
+- The array may be nested more than one level.
+- All values must be integers.
+- Solutions shall not use built in methods to flatten the array to one-dimension.
+- You will need 'rspec' if you want to run the tests. From the project `src/` folder run either:
```
Comment (ericwburden) [inline challenges/2021-04-27-sum-arrays/challenge.md:11]: I might move the system requirements/install instructions to the end, after Examples. Maybe include a link to instructions for installing gem/bundler? (Ruby folks probably have this already).

**Review by ericwburden — CHANGES_REQUESTED** [review]
Minor suggestion for the `challenge.md` file.

File: challenges/2021-04-27-sum-arrays/src/spec/test_array_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  context 'given a two dimension array' do
+    let(:array) { [1, 2, [1, 2, 3], 4, 5] }
+
+    it 'returns 18' do
+      expect(return_value).to eq(18)
+    end
+  end
+
+  context 'givent an n dimension array' do
```
Comment (ericwburden) [inline challenges/2021-04-27-sum-arrays/src/spec/test_array_spec.rb:24]: Typo! ("givent")

Comment (parhaml) [reply challenges/2021-04-27-sum-arrays/src/spec/test_array_spec.rb]: Yikes! Fixed

**Review by ericwburden — APPROVED** [review]

**Review by birenderjit — APPROVED** [review]

**Pull request merged**